Update paragonie/random_compat to 2.x

### Description

Avoid Security Warning by Sensio Security Checker for Flow 3.3 branch

### Steps to Reproduce

Have an flow 3.3 project, create composer.json and either use sensio commandline checker or upload the composer.json here:

* https://security.sensiolabs.org/check

Related Information:

* https://github.com/symfony/symfony/issues/26315

#### Expected behavior

No Security Warning

#### Actual behavior

The following security warning:

```
Symfony Security Check Report
=============================

 // Checked file: /builds/project/composer.lock

 [ERROR] 1 packages have known vulnerabilities.                                                                         

paragonie/random_compat (v1.4.2)
--------------------------------

 * (no CVE ID): Uses insecure CSPRNG (openssl_random_pseudo_bytes())
   https://github.com/paragonie/random_compat/issues/96

 ! [NOTE] This checker can only detect vulnerabilities that are referenced in the SensioLabs security advisories        
 !        database. Execute this command regularly to check the newly discovered vulnerabilities.
```

### Affected Versions

Flow: 3.3.x

You can get this information by running `composer show`.

--- a/TYPO3.Flow/composer.json
+++ b/TYPO3.Flow/composer.json
@@ -28,7 +28,7 @@
         "typo3/eel": "~3.3.0",
 
         "ramsey/uuid": "^3.0.0",
-        "paragonie/random_compat": "^1.0",
+        "paragonie/random_compat": "^2.0",
 
         "doctrine/orm": "~2.5.0",
         "doctrine/migrations": "~1.3.0",

